buildroot/share/fonts/make_fonts.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+bdf2u8g -b  1 -e   9 Marlin_symbols.bdf      Marlin_symbols          dogm_font_data_Marlin_symbols.h
+bdf2u8g -b 16 -e 255 HD44780_W.bdf           HD44780_W_5x7           dogm_font_data_HD44780_W.h
+bdf2u8g -b 32 -e 255 HD44780_C.bdf           HD44780_C_5x7           dogm_font_data_HD44780_C.h
+bdf2u8g -b 32 -e 255 HD44780_J.bdf           HD44780_J_5x7           dogm_font_data_HD44780_J.h
+bdf2u8g -b 32 -e 255 ISO10646-1.bdf          ISO10646_1_5x7          dogm_font_data_ISO10646_1.h
+bdf2u8g -b 32 -e 255 ISO10646-1-tr.bdf       ISO10646_1_tr_5x7       dogm_font_data_ISO10646_1-tr.h
+bdf2u8g -b 32 -e 255 ISO10646_5_Cyrillic.bdf ISO10646_5_Cyrillic_5x7 dogm_font_data_ISO10646_5_Cyrillic.h
+bdf2u8g -b 32 -e 255 ISO10646_Kana.bdf       ISO10646_Kana_5x7       dogm_font_data_ISO10646_Kana.h
+bdf2u8g -b 32 -e 255 ISO10646_CN.bdf         ISO10646_CN             dogm_font_data_ISO10646_CN.h
+bdf2u8g -b 32 -e 255 ISO10646_4_Greek.bdf    ISO10646_Greek_5x7      dogm_font_data_ISO10646_Greek.h
```

Comment:
@yhfudev — This all seems obsolete. Is that correct?

Reply:
Hi, @thinkyhead 
Yes, it should be obsoleted.